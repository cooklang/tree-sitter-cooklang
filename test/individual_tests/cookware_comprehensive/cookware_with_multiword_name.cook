#cast iron skillet{1}