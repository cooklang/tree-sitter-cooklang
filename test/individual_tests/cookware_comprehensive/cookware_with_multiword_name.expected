(recipe
  (step
    (cookware
      name: (cookware_text)
      (amount
        (quantity)))))